src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,83 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+    Sign sign;
+
+    public int calculate(String rpnString){
+        int count = 0;
+        String[] retval = rpnString.split(" ");
+
+         while (retval.length > count){
+            if(retval[count].matches("^[0-9]+$")){
```

Comment:
So think what another useful method you can write in your anum to check if income char is sighn